app-doc/audacity/audacity-9999.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="HTML reference manual for Audacity"
+HOMEPAGE="https://www.audacityteam.org/"
+
+if [[ ${PV} = 9999* ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/audacity/audacity-manual.git"
+else
+	KEYWORDS="amd64 riscv"
+	S="${WORKDIR}/help"
+	SRC_URI="https://github.com/audacity/audacity-manual/releases/download/v${PV}/audacity-manual-${PV}.tar.gz"
+fi
+
+LICENSE="CC-BY-3.0"
+SLOT="0"
+
+DEPEND="!media-sound/audacity[doc]"
```

Comment:
Pure-DEPEND blockers don't mean anything, it needs to be RDEPEND.

But I wonder if we should call it audacity-manual or audacity-doc, despite the category, so people don't end up accidentally emerging the wrong thing. Dunno.

Reply:
The repo is https://github.com/audacity/audacity-manual, so maybe `app-doc/audacity-manual` could be a good candidate.